yaba.yaml

```diff
@@ -0,0 +1,321 @@
+openapi: "3.0.0"
+info:
+  version: 0.0.1
+  title: BETYdb-YABA
+  description: <h2>The YABA API (BETYdb-YABA) is a Standardized RESTful Web Service API Specification for MetaData Upload to respective tables.</h2>
+  license:
+    name: MIT
+servers:
+  - url: http://localhost:8080/yaba/v1
+paths:
+  /experiments:
+    post:
+      summary: Upload metadata to Experiments Table
+      operationId: metaExperiments
+      tags:
+        - experiments
+      requestBody:
+        content:
+          application/json:
+            schema:      # Request body contents
+              type: object
+              properties:
+                name:
+                  type: string
+                start_date:
+                  type: string
+                end_date:
+                  type: string
+                user_id:
+                  type: integer
+              example:   # Sample data
+                name: 'MAC Season 6: Sorghum BAP'
+                start_date: '2018-04-06'
+                end_date: '2018-08-01'
+                user_id: 6000000004
+      responses:
+        '201':
+          description: Successfully Inserted to Experiment Table
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
+
+  /sites:
+    post:
+      summary: Upload metadata to Sites Table
+      operationId: metaSites
+      tags:
+        - sites
+      parameters:
+        - name: shapefile
+          in: query
+          description: 'Shapefile for a site to get geometry'
+          required: true
+          allowReserved: true
+          example: "sometext"
+          schema:
+            type: string
+      requestBody:
```

Comment:
the sites endpoint should take an argument called geometry that accepts either point or polygon geojson

Reply:
There is one parameter which accepts the geojson file in sites endpoint.